boards/arm/nucleo_f756zg/pinmux.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2018 AJ Palmer
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <pinmux.h>
+#include <sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* NUCLEO-F756ZG pin configurations  */
+static const struct pin_config pinconf[] = {
+#ifdef CONFIG_UART_STM32_PORT_2
```

Comment:
Why configuring this one, if not used?

Reply:
Enabled in .dts and device selected by default on CONFIG_SERIAL. 

(I'm using this uart for an offline project. I had overlooked the addition to the branch, this is why the pin definition was present).